src/Dropdown.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from 'styled-system'
+
+import { Box, Flex, Icon, Label } from '../src'
+
+const DropdownFlex = Flex.extend`
+  cursor: pointer;
+  height: 48px;
+  border-radius: ${theme('radius')};
+  border-width: 1px;
+  border-style: solid;
+  border-color: ${theme('colors.borderGray')};
+`
+
+const Dropdown = props => (
+  <DropdownFlex justify="center" align="center">
+    {props.withIcon && (
+      <Box>
+        <Icon name="key" size={24} color="blue" />
```

Comment:
I suspect this would be easier to handle with composition, e.g.:
```
<Dropdown>
  <Icon name='key' />
  Dropdown Label
</Dropdown>
```

Theres a similar discussion here: https://github.com/pricelinelabs/design-system/issues/155

Reply:
@jxnblk, reviewed the concern of #155 and updated the story (`.storybook/Dropdown.js`) to implement a more compound composition. Thanks for the nice feedback. Plz review again.